Changelog.md

```diff
@@ -18,6 +18,9 @@ Compiler Features:
  * Standard-Json: Move the recently introduced ``modelCheckerSettings`` key to ``settings.modelChecker``.
  * Standard-Json: Properly filter the requested output artifacts.
 
+Language Features:
+ * Scanner: Generate a parser error when comments do contain an unbalanced or underflowing set of unicode direction override markers (LRO, RLO, PDF)
```

Comment:
We already have 'Language Features' above :)

Reply:
Also strings.

Reply:
Fixed it now